boards/gd32vf103c-start/Makefile.include

```diff
@@ -0,0 +1,11 @@
+PORT_LINUX ?= /dev/ttyACM0
+PROGRAMMER ?= openocd
+OPENOCD_DEBUG_ADAPTER ?= dap
+OPENOCD_CONFIG := $(RIOTBOARD)/$(BOARD)/dist/openocd.cfg
```

Comment:
```suggestion
```

This is the default anyway

Reply:
It is overwritten in `boards/common/gd32` to one that doesn't work here :-/

Reply:
hm I feel like that is a bit *too* common - maybe move that to the respective boards instead, it does a lot of things that don't feel very generic.

Reply:
OK, I changed it so that boards opt-in to the common OpenOCD config, rather than opt out